tests/foreman/ui/test_myaccount.py

```diff
@@ -127,8 +104,12 @@ def test_positive_update_email(self):
 
         :CaseImportance: Critical
         """
-        email = u'{0}@example.com'.format(gen_string('alpha'))
-        self.assert_text_field_update('users.email', email)
+        with Session(self.browser, self.account_user.login,
+                     self.account_password):
+            email = gen_email()
+            self.my_account.update(email=email)
+            self.my_account.navigate_to_entity()
+            self.my_account.validate_logged_user('email', email)
```

Comment:
I just followed that same approach you implemented on [user](https://github.com/SatelliteQE/robottelo/commit/7a9ae251b8d3534834a4584725c5dbf8359ce82f#diff-e03b60297a285eb104f7e58da16856f1R190). I would also think would be better returning field value, but once conforming to the entire code base was the first reason of all those changes, I am confused on this one ;)